resilience4j-retrofit/README.adoc

```diff
@@ -0,0 +1,39 @@
+= Retrofit Circuitbreaker
+
+https://square.github.io/retrofit/[Retrofit] client circuit breaking
+using
+https://github.com/robwin/javaslang-circuitbreaker[Javaslang-circuitbreaker]
+
```

Comment:
Please consider adding all this documentation to our usage guide in **resilience4j-documentation** module